File: GeeksForGeeks/School/Missing_number.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main() {
	int test_case;
    cin >> test_case;

    while (test_case--) {
        int size;
        cin >> size;

        int i;
        int arr[size - 1];
        for (i = 0; i < size - 1; i++) {
            cin >> arr[i];
        }

        bool status[size];
        memset(status, false, sizeof(status));
        for (i = 0; i < size - 1; i++) {
            status[arr[i] - 1] = true;
        }

        for (i = 0; i < size; i++) {
            if (status[i] == false) {
                cout << i + 1 << endl;
                break;
            }
        }
    }
	return 0;
}

File: GeeksForGeeks/School/Missing_number.java
import java.util.Scanner;

class GFG
 {
	public static void main (String[] args)
	{
	    Scanner scan = new Scanner(System.in);
	    int testcase = scan.nextInt();
	    
    	while (testcase != 0)
    	{
            int size = scan.nextInt();
            int i;
            int[] arr = new int[size];
            for (i = 0; i < size - 1; i++)
            {
                arr[i] = scan.nextInt();
            }

            // Logic Part
            boolean[] status = new boolean[size];
            for (i = 0; i < size - 1; i++)
            {
                status[arr[i] - 1] = true;
            }
            for (i = 0; i < size; i++) {
                if (status[i] == false)
                {
                    System.out.println(i + 1);
                    break;
                }
            }
    	    --testcase;
    	}
    	scan.close();
	}
}